airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -544,6 +544,10 @@ class Tickets(SourceZendeskIncrementalExportStream):
     response_list_name: str = "tickets"
     transformer: TypeTransformer = TypeTransformer(TransformConfig.DefaultSchemaNormalization)
 
+    @staticmethod
+    def check_start_time_param(requested_start_time: int, value: int = 1):
+        return SourceZendeskIncrementalExportStream.check_start_time_param(requested_start_time, value=3)
```

Comment:
Can we add a description why the value is 3 minutes lookback instead of 1? This will help the devs maintain this class in case there is a change made to check_start_time_param

Reply:
Sure. Added description